src/main/kotlin/net/atos/client/smartdocuments/exception/SmartDocumentsBadRequestResponseExceptionMapper.kt

```diff
@@ -13,6 +13,7 @@ class SmartDocumentsBadRequestResponseExceptionMapper : ResponseExceptionMapper<
         status == Response.Status.BAD_REQUEST.statusCode
 
     override fun toThrowable(response: Response): SmartDocumentsBadRequestException = SmartDocumentsBadRequestException(
-        "SmartDocuments responded with a 400 Bad Request: ${response.location} (${response.statusInfo})"
+        "SmartDocuments responded with a 400 Bad Request. " +
+            "Please check if the current user is allowed to interact with SmartDocuments!"
```

Comment:
Is the user the only way to get a bad request? so this feedback will always be valid?

And would it be a good idea to have error pages from backend be styled a little bit? Like now the result page has been done? Quite simple, it there is some templating options. Ofc outside this small PR; but good to have a little thinking about.

Reply:
Every request to SD can return 400 (user missing, malformed parameters, etc.)

This is generic handler to hint that in 90% of the cases users see this error when a user with the same name as in ZAC is not created in SD.

Of course we can also get 400 Bad Request when the SD changes and we no longer comply to the API.

Reply:
OK; let's keep an eye on your assumption 😉